### DISCUSSION TITLE
Klipper compatibility ?

### DISCUSSION THREAD

**Discussion by tocguy** [jneilliii/OctoPrint-BedLevelVisualizer#584] [Q&A]
Transitioned to Klipper and find that this tool doesn't retrieve bed mesh data after selecting "Update Mesh Now" button.
In the plugin settings I also tried it after replacing the Marlin M420 V with Klipper's BED_MESH_OUTPUT PGP=1. Still no joy.

Issuing BED_MESH_OUTPUT PGP=1 in the terminal window produces the expected output.

Plz advise...
Thx

**Comment by jneilliii** [comment]
so this doesn't work? I'm not familiar with the PGP=1 bit since I don't use klipper...

https://github.com/jneilliii/OctoPrint-BedLevelVisualizer/blob/master/wiki/gcode-examples.md#klipper

**Comment by tocguy** [comment]
Nope, but it looks as though the effort to read in the mesh on Klipper would be minimal. On Dec 28, 2022 17:43, jneilliii ***@***.***> wrote:
so this doesn't work? I'm not familiar with the PGP=1 bit since I don't use klipper...
https://github.com/jneilliii/OctoPrint-BedLevelVisualizer/blob/master/wiki/gcode-examples.md#klipper

**Comment by jneilliii** [comment]
well, the klipper commands should be compatible and functioning according to previous issues linked below.

https://github.com/jneilliii/OctoPrint-BedLevelVisualizer/issues/92
https://github.com/jneilliii/OctoPrint-BedLevelVisualizer/issues/264

**Comment by tocguy** [comment]
Roger that. Verified #92 gcode edit restores function in Klipper.On Dec 30, 2022 12:31, jneilliii ***@***.***> wrote:
well, the klipper commands should be compatible and functioning according to previous issues linked below.
#92
#264
